libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/map/FreeDriveMapBinder.kt

```diff
@@ -26,7 +27,12 @@ internal class FreeDriveMapBinder(
                 navigationViewContext.viewModel.cameraState,
                 mapView
             ),
-            RouteFetchComponent(mapView.context),
+            RouteLineComponent(
```

Comment:
Why are adding `RouteLineComponent` to FD?

Reply:
Shouldn't a long click on the map fetch a route and draw it on the map?

Reply:
Yes it should. Free Drive should only be concerned about fetching a route. When it comes to drawing the route line it should happen in `RoutePreview` and `ActiveGuidance` states.

Reply:
The route fetching component listens for map long clicks and when they are received the route(s) are set on MapboxNavigation.  The route line reflects MapboxNavigation. If it has routes they need to be visible on the map.

Reply:
Okay so I am sorry but I was wrong, long pressing on the map should only return the point and we need to use that point to obtain a geocoded address associated with it. It's only when the user decides to either go to route preview or active guidance is when a route needs to be fetched. Hence, the RouteFetchConponent and Route lineComponent should only live in RoutePreviewBinder and ActiveGuidanceBinder